src/DynamoCoreWpf/Views/Search/LibraryView.xaml

```diff
@@ -708,12 +707,12 @@
                                             Property="Foreground"
                                             Value="#AAABAB" />
                                 </Trigger>
-                                <Trigger Property="Button.IsPressed"
-                                         Value="True">
+                                <DataTrigger Binding="{Binding ContextMenu.IsOpen, RelativeSource={RelativeSource Self}}"
```

Comment:
Remove the background color for normal state of the Add button at the bottom.
![image](https://cloud.githubusercontent.com/assets/8158404/6687088/13d8a0ce-ccb0-11e4-81bc-48196454aa46.png)